FWCore/TFWLiteSelectorTest/src/ThingsTSelector.h

```diff
@@ -23,7 +24,8 @@ public :
 
 private:
   /// histograms
-  TH1F * h_a, *h_refA;
+  edm::propagate_const<TH1F*> h_a;
```

Comment:
Would `edm::propagate_const<unique_ptr<TH1F>>` work now? I think we avoided it in the past since ROOT 5 couldn't parse it with cint.